packages/typo3-docs-theme/resources/template/structure/document.html.twig

```diff
@@ -7,12 +7,10 @@
     {{ parent() }}
 {% endblock %}
 
-{% block navbar %}
-    {% include "structure/navigation/navbar.html.twig" %}
-{% endblock %}
-
 {% block menu_doc %}
-    {% include "structure/navigation/menu_doc.html.twig" %}
+    <div aria-label="Main navigation" class="main_menu" role="navigation">
```

Comment:
Instead of `<div>` use `<nav>`? Then the role would be superfluous.

Reply:
Yes I am still working on these kind of changes, but I try to keep the changes small per PR so they are reviewable There is already a nav tag but it seems to be in the wrong position

Reply:
In this case I moved the block without changing it from one template into another